setup.py

```diff
@@ -247,6 +247,13 @@ def sub_commands(self):
         'distutils.commands': [
             'build_sphinx = sphinx.setup_command:BuildDoc',
         ],
+        # consider moving this to 'flake8:local-plugins' once flake8 3.5.0 is
+        # in the wild:
+        #    http://flake8.pycqa.org/en/latest/user/configuration.html\
+        #    #using-local-plugins
+        'flake8.extension': [
+            'X101 = extras.checks:sphinx_has_header',
```

Comment:
Is there any naming conventions? I feel `extras` directory on repos root is a bit strange.

Reply:
I was asked to move this out of `sphinx`, but we don't want to pollute the top-level directory with random Python scripts and seemed strange to put this into `tests`. What would you prefer yourself?

Reply:
@tk0miya I have a request: can we simply include this as `sphinx.checks` for now and remove it as soon as flake8 version (3.5.0) including [the feature I mention in the TODO](http://flake8.pycqa.org/en/latest/user/configuration.html#using-local-plugins) is released? The alternative is package this as a separate package and include that as a dependency, but that seems like overkill :worried:

Reply:
I understand, I don't have no strong opinion. If I had to name it, I will put it on `utils` directory.

Reply:
Sorry, I commented without seeing your second comment.
I much prefer `extra.checks` than `sphinx.checks`. So I think moving to `sphinx` module again is not needed.